### ISSUE TITLE
Multiple allele code compatibility

### ISSUE THREAD

**Issue by ybolon-nmdp** [nmdp-bioinformatics/ngs#146]
Request that both ngs-validate interpretation and ngs-extract-expected be able to expand multiple allele codes in order to cross-validate across results from different platforms.
MHa:"The ngs-validation-report can expand and take in allele codes, but the ngs-validate-interpretation still does not have the capability to expand alleles codes.. 
That means this still needs to be done prior to running that step otherwise it'll always fail. This is an issue that should be added on github, that both ngs-validate-interpretation and ngs-extract-expected need to be able to expand allele codes."

**Comment by mheuer-nmdp** [comment]
I would vote that this not be done in this project since it will have non-open source dependencies.  I think we should start an internal source code repository for validation-related code that cannot be properly open sourced.

**Comment by mheuer-nmdp** [comment]
Closing as WontFix
